subsystems/mvdm/ntvdm/vddsup.c

```diff
@@ -10,15 +10,15 @@
 
 #include "ntvdm.h"
 
-#define NDEBUG
-#include <debug.h>
-
 #include "emulator.h"
 #include "vddsup.h"
 
 #include "cpu/bop.h"
 #include <isvbop.h>
 
+#define NDEBUG
+#include <debug.h>
```

Comment:
Please revert

Reply:
What is wrong with Serges suggestion? I don't see any drawback of including debug always at the end. We do that in many files. Unless you can come up with a specifc problem that would be caused by that in this specific case, I do consider Serges change proposal to be 100% legit.

Reply:
Reverted.